config/host-list.json

```diff
@@ -0,0 +1,12 @@
+{
+"node_label": "label",
+"executors": "",
+"template_detail": {
+"guest_username": "root",
+"guest_password": "passwd",
+"guest_user_credentials_id": "xxxxxxxx"

```

Comment:
i had a VM in mind - Host/Guest concept. when adding a guest(VM) as a node, users can relate to guest_<credential>. I've changed it to node_<credential>. also i've removed guest_password field. not needed